src/main/java/com/minecolonies/coremod/commands/colonycommands/CommandClaimChunks.java

```diff
@@ -59,7 +58,7 @@ public int onExecute(final CommandContext<CommandSource> context)
             return 0;
         }
 
-        if (chunkManager.getAllChunkStorages().size() > CHUNKS_TO_CLAM_THRESHOLD)
+        if (chunkManager.getAllChunkStorages().size() > 5000)
```

Comment:
Well, this should still be a constant, right

Reply:
tbh I don't see the point of a constant if it's only used in one place. But all right, I'll add it into one of the existing constants classes